Tools/DumpRenderTree/mac/TestRunnerMac.mm

```diff
@@ -1214,3 +1218,21 @@ + (void)makeRequest:(NSURLRequest *)request withUsername:(NSString *)username pa
 {
     [[mainFrame webView] _setObscuredTopContentInsetForTesting:top right:right bottom:bottom left:left];
 }
+
+#if ENABLE(DNS_SERVER_FOR_TESTING)
+void TestRunner::initializeDNS()
+{
+    auto agentUUID = adoptNS([[NSUUID alloc] init]);
+    uuid_t agentUUIDBytes;
+    [agentUUID.get() getUUIDBytes:agentUUIDBytes];
+    NSLog(@"useLocalDNSResolver: agent UUID: %@.", agentUUID.get());
+
+    auto webPlatformTestDomain = "web-platform.test"_s;
+    static auto resolverConfig = nw_resolver_config_create();
```

Comment:
Why don't we do `m_resolverConfig = adoptOSObject(nw_resolver_config_create());` here?

Reply:
Good catch! This will be fixed in the upcoming patch.